packages/block-library/src/site-title/index.php

```diff
@@ -20,11 +20,13 @@ function render_block_core_site_title( $attributes ) {
 		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . $attributes['level'];
 	}
 
+	$link = sprintf( '<a href="%1$s" rel="home">%2$s</a>', get_bloginfo( 'url' ), get_bloginfo( 'name' ) );
```

Comment:
Since this is the only `$link` usage we could also just inline the call below.

Reply:
At first, we inlined it but it felt like too much going on in one place. I think it's easier on the eye and brain to keep the link as a variable

Reply:
Yeah, the nested `<a />` and `<%1$s />` (custom tag) were rather confusing, especially since the `sprintf` call then would take a rather long list of args. Splitting into two felt nicer :+1: